examples/twittermap/noah/src/main/resources/twitter/aql/ddl.aql

```diff
@@ -6,6 +6,7 @@ create type typeUser if not exists as open {
     id: int64,
     name: string,
     screen_name : string,
+    profile_image_url : string,
```

Comment:
and let's use this change to remove those misleading index creation as well:

```
//"using" "compaction" "policy" CompactionPolicy ( Configuration )? )?		 //"using" "compaction" "policy" CompactionPolicy ( Configuration )? )?
 create index text_idx if not exists on ds_tweet("text") type keyword;		 create index text_idx if not exists on ds_tweet("text") type keyword;
 create index location_idx if not exists on ds_tweet(coordinate) type rtree;		 create index location_idx if not exists on ds_tweet(coordinate) type rtree;
 create index time_idx if not exists on ds_tweet(create_at) type btree;		 create index time_idx if not exists on ds_tweet(create_at) type btree;
 create index state_idx if not exists on ds_tweet(geo_tag.stateID) type btree;		 create index state_idx if not exists on ds_tweet(geo_tag.stateID) type btree;
 create index county_idx if not exists on ds_tweet(geo_tag.countyID) type btree;		 create index county_idx if not exists on ds_tweet(geo_tag.countyID) type btree;
 create index city_idx if not exists on ds_tweet(geo_tag.cityID) type btree;
```

Reply:
Done

Reply:
sorry, can you change the `keyword` to `fulltext` as well?

Reply:
Done